From ddc5162cc75aeb17716d2dd7f63ae8c274412e94 Mon Sep 17 00:00:00 2001
From: Hanna Hawa <hannah@marvell.com>
Date: Wed, 17 Feb 2016 13:24:14 +0200
Subject: [PATCH 187/538] dts: cp110: add MDIO nodes for Armada-CP-110

- Added SMI MDIO node
- Added xSMI MDIO node

Change-Id: I0cfa33b73fc27454857ed4832a6804c3be44e834
Signed-off-by: Hanna Hawa <hannah@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/27587
Reviewed-by: Yehuda Yitschak <yehuday@marvell.com>
---
 arch/arm64/boot/dts/marvell/armada-cp110.dtsi | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-cp110.dtsi b/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
index ea56a5a..2721a1f 100644
--- a/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
@@ -132,3 +132,17 @@ cpxor1@6c0000 {
 	msi-parent = <&gic_v2m0>;
 	clocks = <&gateclk 7>;
 };
+
+mdio@12a200 {
+	#address-cells = <1>;
+	#size-cells = <0>;
+	compatible = "marvell,orion-mdio";
+	reg = <0x12a200 0x10>;
+};
+
+mdio@15b000 {
+	#address-cells = <1>;
+	#size-cells = <0>;
+	compatible = "marvell,xmdio";
+	reg = <0x12a600 0x10>;
+};
-- 
1.9.1

